Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/PrefabUiHandler.cpp

```diff
@@ -234,6 +242,29 @@ namespace AzToolsFramework
             return;
         }
 
+        if (descendantIndex.column() == EntityOutlinerListModel::ColumnName)
+        {
+            AZ::EntityId descendantEntityId = GetEntityIdFromIndex(descendantIndex);
+
+            // Container entities will always have overrides because they need to maintain unique positions in the scene.
+            // We are skipping checking for overrides on container entities for this reason.
+            if (!m_prefabPublicInterface->IsInstanceContainerEntity(descendantEntityId) &&
+                m_prefabOverridePublicInterface->AreOverridesPresent(descendantEntityId))
+            {
+                // Build the rect that will be used to paint the icon
+                QRect overrideRectangle =
+                    QRect(option.rect.topLeft() + s_overrideIconOffset, QSize(s_overrideIconSize * 2, s_overrideIconSize * 2));
+
+                painter->save();
+                painter->setRenderHint(QPainter::Antialiasing, true);
+                painter->setPen(Qt::NoPen);
+                painter->setBrush(s_overrideIconBackgroundColor);
+                painter->drawEllipse(overrideRectangle.center(), s_overrideIconSize, s_overrideIconSize);
```

Comment:
Is it a rectangle or an ellipse? :)

If it's not painted and just used for sizing, maybe we could call it `overrideIconBounds` or something clearer.